File: src/main/java/com/itri/Hello.java
package com.itri;

public class Hello {

    public static void main(String[] args) {
        /* System.out.println("Hello"); */

        Person p = new Person(67.5f,1.7f);
        p.hello();
        p.Name ="Grey";

        float bmi = p.bmi();
        System.out.println(bmi);

        Person o = new Person("john",68.7f,1.7f);
        System.out.println(o.bmi());



    }
}

File: src/main/kotlin/com/itri/Hello.kt
package com.itri

    fun main(args:Array<String>)
    {
       //println("Hello Kt")
       // Human().hello();
        val h = Human(weight = 66.5f, height = 1.7f)
        h.hello()
        println(h.bmi())
    }

class  Human(var weight:Float,var height:Float,var name:String="") {

    init {
        println("test$weight")
    }

    /*次要建構子*/
//    constructor(name:String,weight: Float,height: Float):this(weight,height);

    fun bmi():Float {

        var bmi = weight / (height*height)
        return  bmi
    }
    fun hello(){

        println("Hello Human")

    }
}